src/xref2/compile.ml

```diff
@@ -53,6 +74,11 @@ and class_type_path : Env.t -> Paths.Path.ClassType.t -> Paths.Path.ClassType.t
       | Ok p' -> `Resolved Lang_of.(Path.resolved_class_type (empty ()) p')
       | Error _ -> p)
 
+let () =
+  (* Until those are used *)
+  ignore value_path;
+  ignore constructor_path
```

Comment:
Would it break things to modify `source_info_infos` like in #976, or would it allow to check more things in the tests?

Reply:
I'm not sure about the question! Initially, #976 was a single PR, but I turned it into three PRs:
- one for the path resolution (this one)
- one for the refactoring #1031
- #976 which will be rebased on top of master when the two other one have been merged.

So no, it won't break things to have `source_infos` as in #976, on the contrary that is the goal!

Reply:
Go it, thanks!